README.md

```diff
@@ -1,96 +1,112 @@
-PLoud Validator version 1.1
-==================
+# PLoud Validator
 
 PLoudValidator is a software to validate audio files with correct loudness.
 It can validate various specifications:
 
 * EBU R 128
-
 * ATSC A85
-
 * CST-Ficam RT-017 specification for France.
 
 It can also correct file to EBU R 128.
 Comming soon for others specifications.
 
-License
--------
+
+## License
 
 See [**COPYING.md**](COPYING.md)
 
-Compilation
------------
 
-* Getting the source
+## Compilation
+
+#### Getting the source
 
-> git clone git://github.com/mikrosimage/loudness_validator.git  
+```
+git clone git://github.com/mikrosimage/loudness_validator.git  
 cd loudness_validator
+```
+
+#### Compilation
+
+###### Build all tools in release mode.
+```
+scons
+```
+
+###### Build in debug mode.
+```
+scons mode=debug
+```
+
+###### Build with a specific path to libraries.
+```
+scons BOOST_ROOT=/path/to/boost SNDFILE_ROOT=/path/to/sndfile
+```
 
-* Linux compilation
 
-> export BOOST_ROOT=path/to/boost  
-export QTDIR=/usr/lib64/qt4  
-./build_linux.sh  
-./build_linux.sh /library /tools /PloudValidator  
+## Run applications
 
-see ./build_linux.sh --help for more informations
+#### Analyser
+```
+build/release/loudness-analyser
```

Comment:
./build/release/loudness-analyser
(or with adding $LOUDNESS_VALIDATOR/build/release to PATH environnement, you can simply run loudness-analyser)